src/main/java/net/minecraftforge/common/world/biomes/conditions/BiomeAndCondition.java

```diff
@@ -0,0 +1,39 @@
+package net.minecraftforge.common.world.biomes.conditions;
+
+import com.mojang.serialization.MapCodec;
+import com.mojang.serialization.codecs.RecordCodecBuilder;
+import net.minecraft.world.biome.Biome;
+import net.minecraftforge.common.world.biomes.ForgeBiomeModifiers;
+import net.minecraftforge.common.world.biomes.conditions.base.BiomeConditionType;
+import net.minecraftforge.common.world.biomes.conditions.base.IBiomeCondition;
+
+public class BiomeAndCondition implements IBiomeCondition
+{
+    public static final MapCodec<BiomeAndCondition> CODEC = RecordCodecBuilder.mapCodec(inst ->
+            inst.group(
+                    IBiomeCondition.INNER_CODEC.fieldOf("first").forGetter(or -> or.first),
+                    IBiomeCondition.INNER_CODEC.fieldOf("second").forGetter(or -> or.second)
```

Comment:
A suggestion: Instead of two fields, use `IBiomeCondtion.INNER_CDEC.listOf()` and `or` together all the elements. That would avoid unnecessary nesting of multiple conditions in json.